This is a testharness.js-based test.
FAIL activating portal inside iframe should fail promise_rejects_dom: function "function() { throw e }" threw an exception from the wrong global
Harness: the test ran to completion.

